src/subscription/azext_subscription/subscription/models/subscription_client_enums.py

```diff
@@ -16,6 +16,8 @@ class OfferType(str, Enum):
 
     ms_azr_0017_p = "MS-AZR-0017P"
     ms_azr_0148_p = "MS-AZR-0148P"
+    MS_AZR_USGOV_0017P = "MS-AZR-USGOV-0017P"
+    MS_AZR_USGOV_0003P = "MS-AZR-USGOV-0003P"
```

Comment:
Shouldn't cases be unified? I.e., all in upper case `MS_AZR_USGOV_0003P` or lower case `ms_azr_0148_p`.